project.Makefile

```diff
@@ -90,15 +87,15 @@ assets/other_mixs_yaml_files/nmdc_mixs_env_triad_tooltips.yaml
 
 # will we ever want to use deepdiff to compare the MIxS schema between two verisons or forms?
 
-project/nmdc_schema_generated.yaml: $(SOURCE_SCHEMA_PATH)
-	# the need for this may be eliminated by adding mandatory pattern materialization to gen-json-schema
-	$(RUN) gen-linkml \
-		--output $@ \
-		--materialize-patterns \
-		--no-materialize-attributes \
-		--format yaml $<
+#project/nmdc_schema_generated.yaml: $(SOURCE_SCHEMA_PATH)
```

Comment:
comment vs remove

Reply:
I should have deleted instead of commenting-out. I will do that in a subsequent PR.